app/javascript/components/upload/SequenceFileStep.jsx

```diff
@@ -43,29 +43,29 @@ function SequenceForm({
     <div className="row">
       <div className="col-md-12">
         <div className="form-terra">
-          Primary sequence information, such as BAM, BAM Index, and FASTQ files<br/>
+          Primary sequence information, such as BAM, BAM Index, and FASTQ files<br />
           <p>
-            <b>Non-human Data</b><br/>
+            <b>Non-human Data</b><br />
```

Comment:
Since it's in the changeset:

```suggestion
            <b>Non-human data</b><br />
```

Reply:
5d5213403